client/state/comments/selectors.js

```diff
@@ -0,0 +1,118 @@
+/***
+ * External dependencies
+ */
+import { createSelector } from 'reselect';
```

Comment:
:sparkles: Also, https://github.com/Automattic/wp-calypso/pull/3522 is open, so whichever PR lands first will probably need to rebase.